Sources/SpeziViews/Views/Text/TextContent.swift

```diff
@@ -8,11 +8,24 @@
 
 import Foundation
 
-
+/// The `TextContent` enum represents either a plain string or a localized string resource.
+///
+/// Use this enum to encapsulate text content in your application, allowing flexibility in handling localized and non-localized strings.
+///
+/// ```swift
+/// // Example usage:
+/// let content: TextContent = .localized(LocalizedStringResource("HELLO_SPEZI", bundle: .main))
+/// let localizedString = content.localizedString(for: .current)
+/// print(localizedString) // Prints the localized string for the current locale.
+/// ```
```

Comment:
Thank you for the added documentation!

The same comments about the example also apply here as well.